tests/test_parsers.py

```diff
@@ -245,3 +246,24 @@ def test_parse_obographs_merged(self):
             )
         msdf = parse_sssom_table(outfile)
         self.assertTrue(custom_curie_map.items() <= msdf.prefix_map.items())
+
+    def test_tsv_to_json_and_back(self):
+        """Test converting SSSOM TSV => JSON => SSSOM TSV such that it is reproducible."""
+        sample_tsv = test_data_dir / "sample1.sssom.tsv"
+        json_outfile = test_out_dir / "sample1.json"
+        msdf1 = parse_sssom_table(sample_tsv)
+        self.assertIn("ORCID", msdf1.prefix_map)
+        msdf1.clean_prefix_map()
+        self.assertIn("ORCID", msdf1.prefix_map)
+        json_doc = to_json(msdf1)
+
+        self.assertEqual(msdf1.prefix_map, json_doc["@context"])
+
+        with open(json_outfile, "w") as file:
+            write_json(msdf1, file)
+
+        msdf2 = parse_sssom_json(json_outfile)
+        self.assertIn("ORCID", msdf2.prefix_map)
```

Comment:
I added some more explicit tests so you can see where things are actually going wrong, which appers to be in the SSSOM JSON parsing, and has nothing to do with cleaning the prefix map like I suggested in a previous comment